{"name": "test_add_product_with_duplicate_id", "status": "broken", "statusDetails": {"message": "sqlite3.IntegrityError: UNIQUE constraint failed: Products.product_id", "trace": "products_table = <dbs.sqlite_products_table.SqliteProductsTable object at 0x104182300>\n\n    def test_add_product_with_duplicate_id(products_table):\n        # Negative test: Attempt to add two products with the same ID\n    \n        # Get the initial count of products in the table\n        initial_product_count = len(products_table.get_all_products())\n    \n        # Attempt to add a product with a random ID\n>       products_table.create_product()\n\ntest_get_products.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nargs = (<dbs.sqlite_products_table.SqliteProductsTable object at 0x104182300>,)\nkwargs = {}, cursor = <sqlite3.Cursor object at 0x104dacf40>\n\n    def wrapper(*args, **kwargs):\n        with SqliteConnector() as cursor:\n>           return cursor.execute(fn(*args, **kwargs)).fetchall()\nE           sqlite3.IntegrityError: UNIQUE constraint failed: Products.product_id\n\n../../dbs/db_base/base_sqlite_table.py:8: IntegrityError"}, "start": 1712645336262, "stop": 1712645336264, "uuid": "7cb476e8-0374-4200-8645-734809ce859f", "historyId": "fad60bd18c5ac601052819b710a6b501", "testCaseId": "fad60bd18c5ac601052819b710a6b501", "fullName": "test.db_tests.test_get_products#test_add_product_with_duplicate_id", "labels": [{"name": "parentSuite", "value": "test.db_tests"}, {"name": "suite", "value": "test_get_products"}, {"name": "host", "value": "dariias-mbp.home"}, {"name": "thread", "value": "17046-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.db_tests.test_get_products"}]}